hack/make-rules/verify/file-perms.sh

```diff
@@ -33,7 +33,7 @@ echo "Checking .sh files permission..."
 # find all files named *.sh (approximate shell script detection ...)
 # - ignoring .git
 # - that are not executable by all
-files=$(find . -type f -name '*.sh' -not -perm "${desired_perm}" -not -path './.git/*')
+files=$(find . -type f -name '*.sh' -not -perm "${desired_perm}" -not -path './.git/*' -path 'node_modules' -path '.bin')
```

Comment:
Do you need to place a `-not` in front of each new `-path` argument?

Also, nit: for many-argument calls, I would prefer to have each of the logical arguments separated out onto its own line with `\` (I think it should work even inside `$(...)`).

Reply:
good catch, fixed